week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,26 @@
+function imageData() {
+  fetch("https://dog.ceo/api/breeds/image/random")
+    .then((response) => {
+      return response.json();
+    })
+    .then((data) => {
+      console.log(data);
+      getImage(data);
+    })
+    .catch((error) => {
+      console.log("Check your internet connection please");
+    });
+}
+
+function getImage(image) {
+  let imageUl = document.querySelector("#list");
+  let imageList = document.createElement("li");
+  let imageBox = document.createElement("img");
+
```

Comment:
I have one suggestion about naming variables. Choosing names is always really hard, sometimes harder than writing the code itself. 😄

It's good practice to give your variables a semantic name, so name it based on what it is, not how you've implemented it.

For example,
`imageUl` - This is the variable that holds your image list. What if I went into your html and changed it from an unordered list <ul> to an ordered list <ol>? Then your variable name would no longer make sense. This can happen a lot in big projects where you work with lots of people and it can make the code become confusing.

I would instead change your variable names to describe what it is on the page.
`imageUl` contains the list of all the images so it would be the `imageList`
`imageList` contains the individual image so it would be `imageItem`
`imageBox` contains the actual image so it would be `image`